templates/patterns/card/pattern-card--variant-search.html.twig

```diff
@@ -4,59 +4,55 @@
  * Card pattern search variant.
  */
 #}
-{% spaceless %}
-  {% set _badges = [] %}
-  {% if badges is not empty %}
-    {% set badge_attr = create_attribute({'class': ['me-2']}) %}
-    {% for key,item in badges %}
-      {% if not loop.last %}
-        {% set badge_attr = create_attribute() %}
-      {% endif %}
-      {% if key|first != '#' %}
-        {% set _badges = _badges|merge([{
-          label: item['#title'],
-          outline: true,
-          rounded_pill: true,
-          attributes: badge_attr
-        }]) %}
-      {% endif %}
-    {% endfor %}
+{% apply spaceless %}
+{% set content %}
+  {% if meta is not empty %}
+    {# Use negative margin to compensate for me-3 on each item. #}
```

Comment:
lets remember to revisit this with BCL 0.21

Reply:
ok but not now, right?